index.js

```diff
@@ -972,6 +972,22 @@
     return either instanceof Left ? l(either.value) : r(either.value);
   });
 
+  //# maybeToEither :: a -> Maybe b -> Either a b
+  //.
+  //. Converts a Maybe value to an Either value, using the supplied
+  //. parameter as the Left value if the Maybe is Nothing.
+  //.
+  //. ```javascript
+  //. > S.maybeToEither('Expecting an integer', S.parseInt(10, 'xyz'))
```

Comment:
> Do I need to comment, or do you as an owner get an email when I push?

I don't get notified when you push, so it's a good idea to comment. At Plaid we `:zap:` pull request comments after addressing them. :)